src/platforms/generic_unix/lib/sys.c

```diff
@@ -550,8 +553,67 @@ Context *sys_create_port(GlobalContext *glb, const char *driver_name, term opts)
 
 term sys_get_info(Context *ctx, term key)
 {
-    UNUSED(ctx);
-    UNUSED(key);
+    if (key == globalcontext_make_atom(ctx->global, ATOM_STR("\x7", "os_type"))) {
```

Comment:
I suggest using `interop_atom_term_select_int` for future-proofing, or anyway no need to make an atom for comparing it, `globalcontext_is_term_equal_to_atom_string` can be used instead.